src/chrome/content/rules/NYDailyNews.xml

```diff
@@ -27,7 +27,7 @@ Fetch error: http://nydailynews.com/ => http://nydailynews.com/: (60, 'SSL certi
 
 	Problematic subdomains:
 
-		- assets *
+		- assets
```

Comment:
Can you move the comment to the `Nonfunctional` section, with a hint about the issue?